File: js/programmers/level.1/prgm_12926_1.js
function solution(s, n) {
  let answer = "";

  for (let i = 0; i < s.length; i++) {
    if (s[i] === " ") {
      answer += " ";
    } else {
      let code = s.charCodeAt(i);
      if (code <= 90) {
        code += n;
        if (code > 90) code -= 26;
      } else {
        code += n;
        if (code > 122) code -= 26;
      }
      answer += String.fromCharCode(code);
    }
  }

  return answer;
}

File: python/programmers/level.1/prgm_12926_1.py
def solution(s, n):
    answer = ""

    for i in s:
        if i == " ": answer += " "
        else:
            code = ord(i)
            if code <= 90:
                code += n
                if code > 90: code -= 26
            else:
                code += n
                if code > 122: code -= 26
            answer += chr(code)

    return answer
